src/components/form-elements/_select.scss

```diff
@@ -97,7 +96,6 @@ $includeHtml: false !default;
 
       .sg-select__element {
         height: $formElementSizeL;
-        line-height: $formElementSizeL - 4px; // subtraction due to top and bottom border
```

Comment:
@mateuszs what if we use percentage value?

Reply:
i observed that it's not needed at all. do you see any need to define it using percentage value?
if it's not set here it gets default value equals `1.5rem` (inherited from `body`).

Reply:
@matzimowski yea, by percentage I meant unit less (which probably is the same :upside_down_face: ), I was wondering why was added but if it works that way we're good!